### PR TITLE
Storybook 8

### PR DESCRIPTION
**Pull request by dartess**
it looks like it just works.
I tested this in examples and in my main project.

Closed https://github.com/hipstersmoothie/storybook-dark-mode/issues/250

### CODE REVIEW

**Comment by dartess** [comment]
I did not change the version in package and did not update the changelog

File: package.json [hunk 1]
```diff
@@ -72,10 +71,10 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "rimraf": "^3.0.2",
-    "storybook": "7.0.0",
+    "storybook": "8.0.0-beta.3",
```
Comment (dartess) [inline package.json:74]: need to update the version after release

File: src/icons/Sun.tsx [hunk 2]
```diff
@@ -9,6 +9,7 @@ export function Sun() {
       enableBackground="new 0 0 129 129"
       style={{
         fill: 'currentColor',
+        height: 13,
```
Comment (dartess) [inline src/icons/Sun.tsx:12]: without this I got a blank icon in the examples.

File: package.json [hunk 3]
```diff
@@ -35,12 +35,11 @@
     "release": "auto shipit"
   },
   "dependencies": {
-    "@storybook/addons": "^7.0.0",
```
Comment (dartess) [inline package.json:38]: @storybook/addons is no longer needed: https://www.npmjs.com/package/@storybook/addons

File: src/icons/Moon.tsx
Comment (dartess) [inline src/icons/Moon.tsx:1]: We can use icons from `@storybook/icons` instead of self-writed. It seems more stable and naturally.

**Comment by bryanjtc** [comment]
@hipstersmoothie Can you review and merge this? 8.0 has been released.

**Comment by hipstersmoothie** [comment]
Just need to get the version updated here @dartess

**Comment by dartess** [comment]
@hipstersmoothie done

**Comment by dartess** [comment]
I just tried to up the version. But I'm not familiar with GitHub workflow.

**Comment by dartess** [comment]
I checked it in my repository - there are no errors, but there are warnings. I'll try to fix them, but it doesn't seem to be critical.

**Comment by dartess** [comment]
@hipstersmoothie
Now everything will be ok, except for [two warnings](https://github.com/dartess/storybook-dark-mode/actions/runs/8254778023).
But it seems it will be safe to update to [update actions](https://github.com/dartess/storybook-dark-mode/commit/b6618bdaba1bc18ebf3b792244877de057dccb59). After this there are [no warnings](https://github.com/dartess/storybook-dark-mode/actions/runs/8254845435).
I can do it here or in a separate PR later, so as not to slow down the release.

**Comment by sshmaxime** [comment]
@dartess I think you are better off doing this in this PR considering it's a pretty small thing. Thanks a lot for your PR 🙏

**Comment by hipstersmoothie** [comment]
Thanks @dartess!

**Pull request merged**